app.py

```diff
@@ -356,6 +367,7 @@ def kyc_incomplete():
 @app.route("/utilities")
 def utilities():
     if check_bronze_oauth(True):
+        add_bronzeuser(g.bronze_userinfo['email'])
```

Comment:
You should change `check_bronze_oauth` to `check_bronze_auth` (as we will first be checking BronzeUser table and then bronze auth)

Move this call into `check_bronze_auth` so that it will not just affect utilities but other stuff later (debit card) too

Reply:
renaming the function and whats being called.